NJW/DAY8/신규아이디추천.js

```diff
@@ -0,0 +1,43 @@
+function solution(new_id) {
+    let answer = "";
+
+    // 1단계 new_id의 모든 대문자를 대응되는 소문자로 치환합니다.
+    let id = new_id.toLowerCase();
```

Comment:
```suggestion
    const id = new_id.toLowerCase();
```

이미 알고계시겠지만, 변하지 않기 때문에 const로 작성하셔도 무방할것 같습니다!

11기 과제 리뷰에서 저는 주로 컨벤션 관점에서 도움이 될만한 것들 위주로 리뷰를 달고 있는데요, 8일차 풀이에서는 이 정도가 끝인것 같습니다. 컨벤션 굿입니다👍

Reply:
감사합니다! let과 const 주의해서 해보겠습니다 :)